SettingsProject/LaunchProfilesWindowViewModel.cs

```diff
@@ -43,7 +43,18 @@ public LaunchProfilesWindowViewModel(ObservableCollection<LaunchProfileViewModel
             
             RenameCommand = new DelegateCommand<LaunchProfileViewModel>(profile => profile.IsRenaming = true);
 
-            NewCommand = new DelegateCommand<LaunchProfileKind>(kind => MessageBox.Show($"New {kind.Name}"));
+            NewCommand = new DelegateCommand<LaunchProfileKind>(kind =>
+            {
+                //TODO: use real data
+                var context = new SettingContext(SettingsLoader.DefaultConfigurationDictionary, LaunchProfilesWindow.Conditions, false, kind.Metadata.Select(md => new Setting(md, new SettingValue(ImmutableDictionary<string, string>.Empty, ""))).ToImmutableArray());
+
+                var newProfile = new LaunchProfileViewModel("New profile", kind, context);
+                
+                Profiles.Add(newProfile);
+                
+                // TODO: select the profile after creating it, implement a notification.
+                //SelectedProfile = newProfile;
```

Comment:
Turns out the lack of notification on `SelectedProfile` resulted in a `NullReferenceException` when using `RenameCommand`. I'll add notification to fix that bug and uncomment this line so that the new profile is selected.